openssl-dynamic/src/main/c/ssl.c

```diff
@@ -891,6 +891,318 @@ TCN_IMPLEMENT_CALL(jint, SSL, sslErrorWantAccept)(TCN_STDARGS) {
     return SSL_ERROR_WANT_ACCEPT;
 }
 
+TCN_IMPLEMENT_CALL(jint, SSL, x509vOK)(TCN_STDARGS) {
```

Comment:
we can move these "static defines" into another file as a followup if you prefer @normanmaurer. I just put them here for now because that is where everything else is defined.